<!DOCTYPE html
  PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
  "DTD/xhtml1-strict.dtd">
<html>
  <head>
    <title>API docs for &ldquo;twisted.python.filepath._WindowsUnlistableError&rdquo;</title>
    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
    <link href="apidocs.css" type="text/css" rel="stylesheet" />
    
    
  </head>
  <body>
    <h1 class="class private">Class t.p.f._WindowsUnlistableError(<a href="twisted.python.filepath.UnlistableError.html">UnlistableError</a>, <span title="twisted.python.win32.WindowsError">WindowsError</span>):</h1>
    <p>
      <span id="part">Part of <a href="twisted.html">twisted</a>.<a href="twisted.python.html">python</a>.<a href="twisted.python.filepath.html">filepath</a></span>
      <a href="http://twistedmatrix.com/trac/browser/tags/releases/twisted-9.0.0/twisted/python/filepath.py#L107">View Source</a>
      <a href="classIndex.html#twisted.python.filepath._WindowsUnlistableError">View In Hierarchy</a>
    </p>
    <div>
      
    </div>
    <div>This exception is raised on Windows, for compatibility with previous 
releases of FilePath where unportable programs may have done &quot;except 
WindowsError:&quot; around a call to children().</p>
<p>It is private because all application code may portably catch <a 
href="twisted.python.filepath.UnlistableError.html"><code>UnlistableError</code></a>
instead.<table class="fieldTable"></table></div>

    
    
    <div id="splitTables">
      
      
      
    </div>
    
    
    

    
    <address>
      <a href="index.html">API Documentation</a> for <a href="http://twistedmatrix.com/">Twisted</a>, generated by <a href="http://codespeak.net/~mwh/pydoctor/">pydoctor</a> at 2009-11-24 22:07:21.
    </address>

    <!-- Google analytics, obviously. -->
    <script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script>
    <script type="text/javascript">
      _uacct = "UA-99018-6";
      urchinTracker();
    </script>
  </body>
</html>